src/App.vue

```diff
@@ -1,30 +1,23 @@
 <template>
   <div id="app">
```

Comment:
Do you need this `div`? You can just `<router-view></router-view>`

Reply:
ok, I thought that root div 'app' is needed